tests/vmcheck/test-autoupdate-stage.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright (C) 2018 Jonathan Lebon
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+
+set -euo pipefail
+
+. ${commondir}/libtest.sh
+. ${commondir}/libvm.sh
+
+set -x
+
+# Prepare an OSTree repo with updates
+vm_ostreeupdate_prepare_reboot
+
+vm_ostreeupdate_create v2
+
+vm_rpmostree cleanup -m
+
+vm_rpmostree status | grep 'auto updates disabled'
+vm_change_update_policy ex-stage
+vm_rpmostree status | grep 'auto updates enabled (stage'
```

Comment:
I think tests are failing because this needs to be:

```
vm_rpmostree status | grep 'auto updates enabled (ex-stage'
```

Actually, let's make try to use `assert_file_has_content` instead? The way it prints the actual output is really useful when looking at test failures.